<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipPackageEditor</class>
 <widget class="QWidget" name="ClipPackageEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clip Package</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">Clips</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="bonusClipTreeWidget">
         <column>
          <property name="text">
           <string>Bonus Clips</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="loadCustomClipsPackagePushButton">
         <property name="text">
          <string>Load Custom Clips</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addClipPushButton">
         <property name="text">
          <string>Add Clip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editClipPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit Clip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeClipPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove Clip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newPackagePushButton">
         <property name="text">
          <string>New Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadPackagePushButton">
         <property name="text">
          <string>Load Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsPushButton">
         <property name="text">
          <string>Save As</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closePackagePushButton">
         <property name="text">
          <string>Close Package</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="../oxygen-icons-png-master/oxygen_windows.qrc">
          <normaloff>:/themes/oxygen/32x32/actions/dialog-close.png</normaloff>:/themes/oxygen/32x32/actions/dialog-close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="icon">
         <iconset resource="../oxygen-icons-png-master/oxygen_windows.qrc">
          <normaloff>:/themes/oxygen/32x32/actions/dialog-ok-apply.png</normaloff>:/themes/oxygen/32x32/actions/dialog-ok-apply.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../oxygen-icons-png-master/oxygen_windows.qrc"/>
 </resources>
 <connections/>
</ui>
